how-to-guides/data-versioning/notebooks/Compare_model_training_runs_on_dataset_versions.ipynb

```diff
@@ -44,19 +44,16 @@
     ]
    },
    "source": [
-    "## Setup"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "tags": [
-     "comment",
-     "installation"
-    ]
-   },
-   "source": [
-    "Install dependencies"
+    "## Before you start\n",
+    "\n",
+    "Make sure that you have:\n",
```

Comment:
Same as previous comment: the default is running the example anonymously through Colab, so Python and registration should not be listed as requirements. They can be mentioned as options

Reply:
Again, here I'm linking to Sabine's Notebook template, specifically "[Before you start](https://colab.research.google.com/drive/1uQgN0uAF74cJXh3v9rmYcdOzUQjkMrIT#scrollTo=FUaDrmRu-h3k)" section. I think that we can take it and adapt here.